### DISCUSSION TITLE
What doc framework does Actix use?

### DISCUSSION THREAD

**Discussion by wietyy** [actix/actix-web#3155] [General] [answered]
Hi! I'm just wondering... I am working on a project I would like to have some docs for, and I think that the docs explaining Actix look pretty. I've seen them on other websites, too. What documentation framework do you guys use?

**Comment by robjtede** [comment, marked as answer]
You can see the code for the website here: https://github.com/actix/actix-website

The framework is called Docusaurus.

**Comment by wietyy** [comment]
Thanks a lot!On Oct 2, 2023 7:33 PM, Rob Ede ***@***.***> wrote:
You can see the code for the website here: https://github.com/actix/actix-website
The framework is called Docusaurus.
